Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8.6-slim-buster AS production
+RUN apt-get update && apt-get install -y curl
+WORKDIR /app
+ENV POETRY_HOME=/poetry
+ENV PATH=${POETRY_HOME}/bin:${PATH}
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+COPY poetry.toml . pyproject.toml .
```

Comment:
This line causes an error when running docker build:
`When using COPY with more than one source file, the destination must be a directory and end with a /`
It's also worth noting that this is copying everything - the COPY instruction allows for multiple src options, but takes a single destination, so this is copying `poetry.toml`, all not-ignored files (`.`) and then `pyproject.toml` explicitly all to the work directory (final `.`). I suspect that's a confusion; was this meant to just copy the toml files?